frontend/src/components/GrantsIdentifier.tsx

```diff
@@ -0,0 +1,106 @@
+import { Trans, useTranslation } from "next-i18next";
+import Image from "next/image";
+import {
+  Identifier,
+  IdentifierGov,
+  IdentifierIdentity,
+  IdentifierLink,
+  IdentifierLinkItem,
+  IdentifierLinks,
+  IdentifierLogo,
+  IdentifierLogos,
+  IdentifierMasthead,
+} from "@trussworks/react-uswds";
+
+import logo from "../../public/img/logo-white-lg.webp";
+
+const GrantsIdentifier = () => {
+  const { t } = useTranslation("common", {
+    keyPrefix: "Identifier",
+  });
+
+  const logoImage = () => {
+    return (
+      <Image
+        alt={t("logo_alt")}
+        src={logo}
+        height={48}
+        width={48}
+        style={{
+          width: "48px",
+          height: "48px",
+        }}
```

Comment:
@andycochran The image I downloaded from here https://www.hhs.gov/sites/default/files/logo-white-lg.png as a webp file is 1200/1200 px, so that's overwriting the 3rem style set by `.usa-identifier__logo-img`. With my mac, I don't seem to be able to convert the size of a webp file, or convert a png file into webp. Can you help me with that?

Reply:
The image itself needs the `.usa-identifier__logo-img` class.

Reply:
The `3rem` is on the image container. The `.usa-identifier__logo-img` when applied to the image sets it to fill that container (`height:100%` and `width:auto`). There's no need to set an explicit pixel width for the image. Doing so isn't recommended because users settings can zoom pixels differently than `rem` on some browsers.

Reply:
```suggestion
        className="usa-identifier__logo-img"
```

Reply:
Oh, that's really helpful! Thanks for going deep and explaining. I was assuming the Truss IdentifierLogo would add that, but it doesn't.